packages/core/src/components/RadioButton/RadioButton.tsx

```diff
@@ -64,6 +66,7 @@ const RadioButton: VibeComponent<RadioButtonProps, HTMLElement> & object = forwa
        */
       radioButtonClassName,
       disabled = false,
+      autoFocus = false,
```

Comment:
maybe leave it `undefined` in case it isn't supplied
so current snapshots won't add `autoFocus=false` to their snapshots, as it can lead to - when upgrading a version in monolith - to multiple snapshots being broken
we're ok with it as `undefined`
so just remove the default `false`

Reply:
Removed :)